client/components/admin/AdminTeams/imports/AdminTeamModalProgress.jsx

```diff
@@ -0,0 +1,50 @@
+import { Meteor } from 'meteor/meteor';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { Table } from 'semantic-ui-react';
+
+class AdminTeamModalProgress extends Component {
+  render() {
+    const { team } = this.props;
+    const { puzzles } = team;
+
+    const puzzlesCt = puzzles.length;
+    const puzzlesComplete = puzzles.reduce((acc, puzzle) => {
+        return acc + ( puzzle.end ? 1 : 0 );
+    }, 0);
+
+    return (
+        <div>
+            {team.hasBegun ? "" : "Team hasn't started game (yet)"}
```

Comment:
Decrease indent through line 41

Reply:
I'm not quite sure I follow.... the indentation looks right on my screen, unless I'm missing something painfully obvious